react-native-git-upgrade/cli.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+require('babel-register')({
+  presets: [
+    require('babel-preset-es2015-node'),
+    require('babel-preset-stage-3')
```

Comment:
Is this for async & await? Since we'll also be merging the code in react-native-git-upgrade directly into the RN CLI we'd have to add this preset there as well, correct?

Do we transpile the CLI code with Babel at all currently? If not I'd prefer to keep things simple (run it with node as-is) and would prefer to use Promises rather than async await. The code becomes a bit more verbose but hopefully not that much, chaining Promises with `then`?